lib/Doctrine/Common/Collections/ExpressionBuilder.php

```diff
@@ -40,19 +40,19 @@ class ExpressionBuilder
      *
      * @return CompositeExpression
      */
-    public function andX($x = null)
+    public function andX(...$x)
```

Comment:
@stof are you sure about strings? https://github.com/doctrine/collections/blob/0b0b54166319e02f50f996f8d966aed368a78bfe/lib/Doctrine/Common/Collections/Expr/CompositeExpression.php#L49 seems to say something different...